libs/blocks/global-navigation/utilities/keyboard/index.js

```diff
@@ -25,6 +25,47 @@ const cycleOnOpenSearch = ({ e, isDesktop }) => {
   }
 };
 
+const openProfile = ({ e, el }) => {
+  const button = e.target.closest(`${selectors.signIn}, ${selectors.profileButton}`);
+  if (button && button.getAttribute('aria-expanded') === 'false') {
+    e.target.click();
+    el.querySelector(selectors.profileDropdown).focus();
```

Comment:
```suggestion
    el.querySelector(selectors.profileDropdown)?.focus();
```
just to be extra safe